recipe/0002-remove-linker-flags.patch

```diff
@@ -0,0 +1,29 @@
+From 17f7d963f6d9717699c320f4b4ea5a2b47698eae Mon Sep 17 00:00:00 2001
+From: Jack Olivieri <jolivieri@anaconda.com>
+Date: Fri, 3 Nov 2023 20:34:47 +0100
+Subject: [PATCH] remove linker flags
+
+---
+ setup.py | 6 ------
+ 1 file changed, 6 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index 37c87c4..6837667 100755
+--- a/setup.py
++++ b/setup.py
+@@ -229,12 +229,6 @@ def ext(name: str,
+             }.get(system, ()),
+             *extra_compile_args
+         ],
+-        extra_link_args=[
+-            *{
+-                'Linux': ('-Wl,-rpath,$ORIGIN/lib/Linux',),
```

Comment:
I suppose we do not need to provide: $LD_RUN_PATH
which in my env is set to  /home/mwaszkiewicz/miniconda3/conda-bld/debug_1699521449465/_h_env/lib, because it is automatically added by conda ?

Reply:
`$PREFIX/lib` is always included by Anaconda by default as a searchable path for libraries.

Reply:
the `$RPATH` that they included here is simply bogus because that path does not exist.